### ISSUE TITLE
tfa.losses.TripletSemiHardLoss() replacement?

### ISSUE THREAD

**Issue by mfide** [tensorflow/similarity#370]
Hello,

I was using triplet loss function from `tensorflow_addons.losses.TripletSemiHardLoss(margin=1.0)` to train my model using following line:

`tsn.compile(optimizer=tf.keras.optimizers.Adam(0.0001), loss=tfa.losses.TripletSemiHardLoss(margin=1.0))
`

Any example or idea how to replace this loss function with tripletloss in `tensorflow_similarity.losses`?

Thank you.

**Comment by mhyeonsoo** [comment]
I am also wondering about this...

**Comment by ma7555** [comment]
`tfsim.losses.TripletLoss(distance='l2', positive_mining_strategy='hard', negative_mining_strategy='semi-hard', margin=1.0)`
